pyproject.toml

```diff
@@ -22,8 +22,23 @@ pytest = "^7.2.2"
 black = "^23.1.0"
 isort = "^5.12.0"
 pylint = "^2.17.0"
+poethepoet = "^0.19.0"
 
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.poe.tasks]
+
+    [tool.poe.tasks.format-isort]
+    help = "Run isort on the code base"
+    cmd  = "isort ."
+
+    [tool.poe.tasks.format-black]
+    help = "Run black on the code base"
+    cmd  = "black ."
+
+    [tool.poe.tasks.format]
+    help = "Run formating tools on the code base"
+    sequence  = ["format-isort", "format-black"]
```

Comment:
We could probably run `pylint` as well, couldn't we?

Reply:
I wouldn't add `pylint` to the `format` command, because a linter and a formatter are two different things 😇 We could add a new command `prepare-push` and run `poe format` and `pylint`.